deployer/models/release_resources.go

```diff
@@ -90,7 +90,9 @@ func (release *Release) FetchResources(asgc aws.ASGAPI, ec2 aws.EC2API, elbc aws
 		resources.ServiceResources[name] = sr
 	}
 
-	release.WaitForDetach = &slowStartDuration
+	if release.WaitForDetach == nil || release.WaitForDetach == 0 {
```

Comment:
technically the nil check is unnecessary as WaitForDetach is set in defaults, but i thought it's safer to check anyway